submit_flagged.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+	require_once($_SERVER["DOCUMENT_ROOT"] . "/blaze/base.php");
+	
+	$validation = validateToken($_POST["access_token"], $_SERVER["REMOTE_HOST"]);
+	if($validation[0] !== 200) {
+		http_response_code($validation[0]);
+		echo json_encode(array("error_code" => $validation[1], "error_message" => $validation[2]));
+		exit;
+	}
+	
+	$conn = get_db_connection();
+	
+	$post_text = mysqli_real_escape_string($conn, substr($_POST['post_text'], 0, 40000));
+	
+	if(!isset($_POST['post_text']) || $post_text == '') {
+		// #b.404/C: No post text provided to add to database.
+		http_response_code(400);
+		echo json_encode(array('error_code' => 'b.404+C', 'error_message' => 'No post text provided to add to database.'));
+		exit;
+	}
+	else {
+		$result = mysqli_query($conn, 'INSERT INTO Posts VALUES (DEFAULT, DEFAULT, \'' . $post_text . '\');');
```

Comment:
Current table name is actually `posts` in MySQL, not sure if it's case sensitive or not.